src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -838,11 +838,22 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
     {
-        $container
+        $definition = $container
             ->getDefinition('property_accessor')
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['cache']) && $config['cache']) {
+            $container->setParameter(
+                'property_accessor.cache.prefix',
```

Comment:
That is used multiple times shows one more reason to move it to doctrine-cache-bundle.